src/types.ts

```diff
@@ -41,6 +43,7 @@ export interface ImageData {
   data: PngDataArray;
   depth?: BitDepth;
   channels?: number;
+  interlace?: InterlaceMethod;
```

Comment:
This is not a property of the image data and should be passed as a option.
We should make the option more user-friendly also (a string like 'null' | 'Adam7').